src/code/z_construct.c

```diff
@@ -373,10 +373,12 @@ void Regs_InitDataImpl(void) {
     R_START_LABEL_X(2) = 119;
 #endif
 
-    ZREG(61) = 1;
-    R_C_UP_BTN_X = C_UP_BUTTON_X;
-    R_C_UP_BTN_Y = C_UP_BUTTON_Y;
-    ZREG(64) = 20;
+    R_PAUSE_QUEST_MEDALLION_SHINE_TIME(0) = 1;
+    //! @bug Overlapping reg usage
+    R_C_UP_BTN_X = C_UP_BUTTON_X; // R_PAUSE_QUEST_MEDALLION_SHINE_TIME(1)
+    R_C_UP_BTN_Y = C_UP_BUTTON_Y; // R_PAUSE_QUEST_MEDALLION_SHINE_TIME(2)
```

Comment:
Is it truly a bug, like do values conflict? I could see them reusing regs in a smart way if they felt like they were running out space in their assigned reg letter or something

Reply:
It does seem like the values conflict to me, but I'm kinda curious of the medallion animation is off as a result